lib/opensusebasetest.pm

```diff
@@ -336,11 +333,6 @@ sub wait_boot {
             # harddisk' is above
             send_key_until_needlematch 'inst-bootmenu-boot-harddisk', 'up';
             boot_local_disk;
-            if (check_var('ARCH', 'aarch64') and get_var('UEFI')) {
```

Comment:
well I figured that the problem we are trying to solve for aarch64 is the same. bootindex is not honored or not properly working and we need to do some additional steps. Sure, they differ for aarch64 and ppc64le but the net result is the same: you end up in the boot menu on the local disk.